.github/workflows/ci-casper-client-rs.yml

```diff
@@ -45,6 +53,13 @@ jobs:
           command: clippy
           args: --all-targets
 
+      # // Awaits https://github.com/casper-ecosystem/casper-client-rs/pull/153
+      # - name: Clippy with no features
+      #   uses: actions-rs/cargo@v1
+      #   with:
+      #     command: clippy
+      #     args: --all-targets --no-default-features
```

Comment:
I think we should include the existing commands, just without all targets for now, and add --all-targets to the wasm PR

Reply:
Ok but those are already on dev branch https://github.com/casper-ecosystem/casper-client-rs/blob/dev/.github/workflows/ci-casper-client-rs.yml

and the feature it self is already on feat-track-2.0, it will still fail until the PR is merged so I commented out.

it is more confusing to repeat the every command twice and change dev version which is correct.

the version uncommitted is still not correct imo. https://github.com/casper-ecosystem/casper-client-rs/pull/185/commits/bd9d4f591a0cdafdc538f5e3b700e72296ff04b6

Reply:
the commands pass on my machine